src/execution_impl.rs

```diff
@@ -22,16 +22,14 @@ use wasmer_middlewares::Metering;
 /// this environment is automatically filled by the wasmer library
 /// And two pointers of string. (look at the readme in the wasm folder)
 fn call_module(env: &Env, address: &Address, function: &str, param: &str) -> Result<Response> {
-    let get_module: fn(&Address) -> Result<Bytecode> = env.interface.get_module;
-    sub_remaining_point(env, settings::METERING.call_price()).unwrap();
```

Comment:
Hey @gterzian, I think that the metering problem that you have is that you removed this line.

Reply:
looks like you're right, thanks!